service/worker/common/interface.go

```diff
@@ -58,6 +58,14 @@ type (
 		Options sdkworker.Options
 	}
 
+	ActivityWorkerConcurrencyConfig struct {
```

Comment:
nit: looks like it's more than just concurrency, maybe rate limit or load config?

Reply:
Is a rate limit not a form of concurrency control? But sure I can change it